action.yml

```diff
@@ -67,13 +73,13 @@ runs:
       run: |
         if npm run | grep -q "test:unit" ; then
           echo "Unit test script found"
-          npm run test:unit
+          ${{ inputs.unit-test-command }}
         else
           echo "No unit tests defined, skipping..."
         fi
 
     - name: Run integration tests
       shell: bash
-      run: npm run test:integration
+      run: ${{ inputs.integration-test-command }}

```

Comment:
If we do the above, we should be consistent and do the same with the integration test command.

Reply:
The `test:unit` part of the condition can now also be overwritten using the input `unit-test-check-pattern`.